### PR TITLE
Added metadata to support docs generation

### PR DESCRIPTION
**Pull request by AlbertGromek**
## Description
Added x-examples for runnable code samples in the generated docs.

## Types of Changes
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: xero-payroll-au.yaml [hunk 1]
```diff
@@ -875,6 +1189,252 @@ paths:
...
+          is_last: true 
+          default: 850            
+        - earningsRates:
+          is_list: true 
+          key: earningsRates
+          keyPascal: EarningsRates
+          keySnake: earnings_rates
+          csharp: EarningsRate
+          java: EarningsRate
+        - add_earningsRates:
```
Comment (SerKnight) [inline xero-payroll-au.yaml:1332]: should these param names be half snake? Don't think i've seen that anywhere else 🤔

Comment (AlbertGromek) [reply xero-payroll-au.yaml:1332]: @SerKnight 

image

I actually found 88 occurences in xero_accounting.yaml
Did we just invent half snake case?

Looks like the naming convention for those params seems to be camelCase in general so I'll go ahead and change those in payroll-au and accounting

**Review by RettBehrens — APPROVED** [review]

**Review by SerKnight — APPROVED** [review]
Fantasmic

**Pull request merged**